odoh_test.go

```diff
@@ -202,19 +203,18 @@ func TestConfigsDeserialization(t *testing.T) {
 }
 
 func createDefaultSerializedPublicKey(t *testing.T) []byte {
-	suite, err := hpke.AssembleCipherSuite(ODOH_DEFAULT_KEMID, ODOH_DEFAULT_KDFID, ODOH_DEFAULT_AEADID)
-	if err != nil {
-		t.Fatalf("Failed generating HPKE suite")
-	}
+	suite := hpke.NewSuite(ODOH_DEFAULT_KEMID, ODOH_DEFAULT_KDFID, ODOH_DEFAULT_AEADID)
 
-	ikm := make([]byte, suite.KEM.PrivateKeySize())
+	kemScheme := suite.KemID.Scheme()
+	ikm := make([]byte, kemScheme.PrivateKeySize())
 	_, _ = rand.Read(ikm)
```

Comment:
Unrelated to your change, but can we handle this error?

Reply:
done